source/principles/general/index.html.md

```diff
@@ -0,0 +1,35 @@
+---
+last_reviewed_on: 2022-01-07
+review_in: 3 months
+weight: 5
+---
+
+# General principles
+
+These pages lay out principles for architecture, coding, and developing frontends for DfE services. On this page, we lay
+out more generic principles for technical folk at the department.
+
+## Make things open, it makes things better
+All of the code that we produce is classified at OFFICIAL and is paid for by the taxpayer. We have a duty to make that 
+code open, to encourage reuse, and in doing so reduce waste. That's why we use GitHub to store our code, unless there's 
+a security or policy reason to keep our work secret. Even the spooky folks at the Government Communications Headquarters
+(GCHQ) have published a number of [open-source repos](https://github.com/gchq). [We've written guidance on how to use our 
+preferred tool, GitHub](../../standards/storing-source-code/index.html.md.erb)
+
+## Continuous integration
```

Comment:
This overlaps with https://technical-guidance.education.gov.uk/guides/continuous-delivery/
Could we link to it as least?

Reply:
Yeah, good shout!